core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java

```diff
@@ -530,6 +531,17 @@ public void invoke() {
         }
     }
 
+    // #460 : docker-compose are not resolvable on Win10 machines because there is docker-compose.exe binary,
+    // but not docker-compose
+    private boolean isDockerComposeExists() {
```

Comment:
What if we change this method to:

```
        if (SystemUtils.IS_OS_WINDOWS) {
            return CommandLine.executableExists(COMPOSE_EXECUTABLE + ".exe");
        } else {
           return CommandLine.executableExists(COMPOSE_EXECUTABLE);
        }
```

Reply:
This variant more readable, but narrows all possible options to *.exe variant on Win machines. 
How docker-compose executable looks on other Win platforms (not Win10) ?

Reply:
Since we currently only support Docker for Windows on Windows 10 (see [Windows support](https://www.testcontainers.org/usage/windows_support.html)), I would be fine with this.

Reply:
In this case I will rewrite it.